src/main/resources/templates/api.html

```diff
@@ -1,188 +1,215 @@
 <!DOCTYPE html>
 <html xmlns:th="http://www.thymeleaf.org">
+
 <head>
     <title th:text="${api.id} != null ? 'Editar API' : 'Cadastro de API'"></title>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css">
+    <link href="https://bootswatch.com/5/cerulean/bootstrap.min.css" rel="stylesheet">
+    <style>
+        body{
+            font-size: 14px;
+        }
+    </style>
 </head>
+
 <body>
-<nav class="navbar navbar-expand-lg navbar-light bg-light">
-    <a class="navbar-brand" href="#">Projeto</a>
-    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-    </button>
-    <div class="collapse navbar-collapse" id="navbarNav">
-        <ul class="navbar-nav">
-            <li class="nav-item active">
-                <a class="nav-link" th:href="@{/index}">Consulta de Notícias<span class="sr-only">(atual)</span></a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" th:href="@{/portais}">Cadastro de Portal</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" th:href="@{/apis}">Cadastro de API</a>
-            </li>
-        </ul>
-    </div>
-</nav>
+    <nav class="navbar bg-dark border-bottom border-body" data-bs-theme="dark">
+        <a class="navbar-brand" href="#" style="margin-left: 10px;">
+            <svg xmlns="http://www.w3.org/2000/svg" width="22" height="22" fill="currentColor" class="bi bi-newspaper"
+                viewBox="0 0 16 16">
+                <path
+                    d="M0 2.5A1.5 1.5 0 0 1 1.5 1h11A1.5 1.5 0 0 1 14 2.5v10.528c0 .3-.05.654-.238.972h.738a.5.5 0 0 0 .5-.5v-9a.5.5 0 0 1 1 0v9a1.5 1.5 0 0 1-1.5 1.5H1.497A1.497 1.497 0 0 1 0 13.5zM12 14c.37 0 .654-.211.853-.441.092-.106.147-.279.147-.531V2.5a.5.5 0 0 0-.5-.5h-11a.5.5 0 0 0-.5.5v11c0 .278.223.5.497.5z" />
+                <path
+                    d="M2 3h10v2H2zm0 3h4v3H2zm0 4h4v1H2zm0 2h4v1H2zm5-6h2v1H7zm3 0h2v1h-2zM7 8h2v1H7zm3 0h2v1h-2zm-3 2h2v1H7zm3 0h2v1h-2zm-3 2h2v1H7zm3 0h2v1h-2z" />
+            </svg>
+        </a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav"
+            aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav">
+                <li class="nav-item active">
+                    <a class="nav-link" th:href="@{/index}">Consulta de Notícias</a>
```

Comment:
![image](https://github.com/user-attachments/assets/61811643-d497-4d45-a9ec-aa81a20b4fcd)
Esse ícone não está clicável, acho que pode confundir o cliente. Deixa só essa parte com 3 risquinhos, se possível, trás os links navegáveis pro lado direito. Ou, trás esses 3 traços pra esquerda e deixa a escrita na esquerda mesmo, o que for mais fácil.
![image](https://github.com/user-attachments/assets/b4a69abc-7197-41b6-a6f4-bf587d4b6fdd)